public/docs/_examples/toh-6/ts/app/hero-detail.component.ts

```diff
@@ -29,11 +31,11 @@ export class HeroDetailComponent implements OnInit {
   // #docregion save
   save(): void {
     this.heroService.update(this.hero)
-      .then(this.goBack);
+      .then(() => this.goBack());
```

Comment:
I was getting an error before that it `Cannot call location of null` error. Maybe it was a fluke on my part but I just tried it again and it worked correctly. I'll change it back to what it was.